### DISCUSSION TITLE
Diff support for dataset runs view

### DISCUSSION THREAD

**Discussion by saikatmitra91** [langfuse/langfuse#4025] [Ideas]
### Describe the feature or potential improvement

For the dataset runs, a lot of time we end up reviewing the output side by side with the expected output.

When the outputs are structured data, then its very tedious to find the difference between output and expected output. 

What can help here is to get a diff view supported so that I can quickly browse the difference between expected and the current output. This will really boost productivity in human reviews.

**Comment by marcklingen** [comment]
This is available now, forgot to update this thread: https://langfuse.com/docs/datasets/get-started#analyze-dataset-runs
